### PR TITLE
Added UC features pages & updated page's content

### PR DESCRIPTION
**Pull request by zprood**
## Done

1. Added and translated features pages of Ubuntu Core from [](https://ubuntu.com/core/features) including:
- Features
- Secure boot
- Full disk encryption
- Recovery
- OTA updates
2. Creating a submenu for features pages.
3. Updated snap store screenshot of the internet-of-things page.
4. Updated Contact page text. 

## QA

- [ ] Features page submenu

### CODE REVIEW

File: templates/internet-of-things/features/recovery.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+      </div>
+      <div class="col-7">
+        <h2>低接触设备恢复</h2>
+        <p>与Ubuntu安全专家联系，讨论您应用程序的高级安全要求。</p>
+        <p>
+          <a href="/contact" class="p-button--positive js-invoke-modal">Get in touch</a>
```
Comment (anthonydillon) [inline templates/internet-of-things/features/recovery.html:130]: This should be translated. If this section is the same on all pages you could copy the section into a partial file and include there with:
```
{% include "shared/_contact-strip.html" %}
```
_Files that begin with `_` indicates its a partial file._

File: templates/internet-of-things/features/secondary-nav.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <a class="breadcrumbs__link {% if request.path == '/internet-of-things/features/full-disk-encryption' %}p-link--active{% else %}p-link--soft{% endif %}" href="/internet-of-things/features/full-disk-encryption">全盘加密</a>
+          </li>
+          <li class="breadcrumbs__item">
+            <a class="breadcrumbs__link {% if request.path == '/internet-of-things/features/recovery' %}p-link--active{% else %}p-link--soft{% endif %}" href="/internet-of-things/features/recovery">恢复模式</a>
+          </li> 
+          <li class="breadcrumbs__item">
+            <a class="breadcrumbs__link {% if request.path == '/internet-of-things/features/ota-updates' %}p-link--active{% else %}p-link--soft{% endif %}" href="/internet-of-things/features/ota-updates">OTA更新</a>
+          </li>
+          <li class="breadcrumbs__item">
+            <a class="breadcrumbs__link" href="https://ubuntu.com/pricing/devices">服务和价格</a>
```
Comment (anthonydillon) [inline templates/internet-of-things/features/secondary-nav.html:26]: Missing `class="p-link--soft"`

File: templates/internet-of-things/features/secondary-nav.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<nav class="p-navigation--secondary">
+    <div class="row">
+      <div class="col-12 u-equal-height">
+        <a class="p-navigation--secondary__banner u-hide--nav-threshold-down" href="/internet-of-things/features">
```
Comment (anthonydillon) [inline templates/internet-of-things/features/secondary-nav.html:4]: This one should link back to `/internet-of-things`

**Review by anthonydillon — APPROVED** [review]
@zprood Great work, works well. Few small comments.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Comments in code.

File: templates/internet-of-things/features/secondary-nav.html (hunk 2 above)
Comment (zprood) [reply templates/internet-of-things/features/secondary-nav.html]: Done.

File: templates/internet-of-things/features/secondary-nav.html (hunk 3 above)
Comment (zprood) [reply templates/internet-of-things/features/secondary-nav.html:4]: Done.

File: templates/internet-of-things/features/recovery.html (hunk 1 above)
Comment (zprood) [reply templates/internet-of-things/features/recovery.html]: Hi Anthony, currently we don't have enough pages for different forms based on pages, so I remove the shared form quote. That's the next step of the plan. Thanks a lot.

File: templates/internet-of-things/features/secondary-nav.html (hunk 3 above)
Comment (anthonydillon) [reply templates/internet-of-things/features/secondary-nav.html:4]: I don't think this one is done yet. If you go to https://cn-ubuntu-com-530.demos.haus/internet-of-things/features/full-disk-encryption and click the "Core特点" in the second level navigation it should take you back to /internet-of-things.

**Review by anthonydillon — APPROVED** [review]
Looks good other than the comment in the code around the second level nav.

File: templates/internet-of-things/features/secondary-nav.html (hunk 3 above)
Comment (zprood) [reply templates/internet-of-things/features/secondary-nav.html]: Oh sorry, I forget to update this one but changed it in my local environment. Now updated. THX.

**Pull request merged**